api/arceos_api/src/imp/task.rs

```diff
@@ -82,6 +82,17 @@ cfg_task! {
         }
     }
 
+    pub fn ax_set_current_affinity(cpumask: axtask::CpuMask) -> crate::AxResult {
```

Comment:
Use type alias `AxCpuMask`, we don't want to import types from other crates.

Reply:
type alias:
```rust
/// The wrapper type for [`cpumask::CpuMask`] with SMP configuration.
pub type AxCpuMask = cpumask::CpuMask<{ axconfig::SMP }>;
```

I alse expose it in arceos_api like this:
```rust
    define_api_type! {
        @cfg "multitask";
        pub type AxTaskHandle;
        pub type AxWaitQueueHandle;
        pub type AxCpuMask;
    }

    /// A mask to specify the CPU affinity.
    pub use axtask::AxCpuMask;
```